components/catalogs/front/src/components/ProductSelection/criteria/AttributeYesNoCriterion/factory.ts

```diff
@@ -0,0 +1,16 @@
+import {AttributeYesNoCriterionState} from './types';
+import {Operator} from '../../models/Operator';
+
+export default (state?: Partial<AttributeYesNoCriterionState>): AttributeYesNoCriterionState => {
+    if (!state?.field) {
+        throw Error('You need to specific the attribute code when calling the attribute criterion factory');
+    }
+
+    return {
+        field: state.field,
+        operator: state?.operator ?? Operator.EQUALS,
+        value: state?.value ?? null,
```

Comment:
Either the default operator is empty or the default value is a boolean
```suggestion
        value: state?.value ?? false,
```

Reply:
I would say that true should be the default value, not false.
But @meganelepalud-akeneo should decide.